### DISCUSSION TITLE
Is it possible to tell autocomplete to switch to more generic autocomplete?

### DISCUSSION THREAD

**Discussion by Maykeye** [fish-shell/fish-shell#10939] [Q&A]
Example: If I type `pri<tab>`, fish offers "primes", "prime-run", etc. It doesn't however offer `org.prismlauncher.PrismLauncher`. If I type `pris<tab>`, it does. There are no programs that start with `pris`, so it tries harder.
How to make it offer "org.prismlauncher.PrismLauncher" as well like it does if I type `pri<tab>`?

(I'm not interested to have this generic autocomplete all the time, only on occasions like prism or  `freecad`  vs `FreeCAD`)

**Comment by krobelus** [comment]
`org.prismlauncher.PrismLauncher` is not a valid command so it should not be suggested in command position.
It is suggested on commands where it is valid, for example `xdg-mime default `, courtesy of [xdg-mime.fish](https://github.com/fish-shell/fish-shell/blob/master/share/completions/xdg-mime.fish)
